spinalcordtoolbox/reports/slice.py

```diff
@@ -158,10 +159,10 @@ def add_slice(matrix, i, column, size, patch):
         :param patch: patch to insert
         :return: matrix
         """
-        start_col = (i % column) * size * 2
+        start_col = (i % column) * patch.shape[1]
         end_col = start_col + patch.shape[1]
 
-        start_row = int(i / column) * size * 2
+        start_row = int(i / column) * patch.shape[0]
```

Comment:
Unlike `math.ceil` used elsewhere, I think this _should_ use floor division:
```suggestion
        start_row = (i // column) * patch.shape[0]
```